processing/src/test/java/org/apache/druid/query/topn/TopNQueryRunnerTest.java

```diff
@@ -4541,6 +4541,79 @@ public void testFullOnTopNLongColumn()
     assertExpectedResults(expectedResults, query);
   }
 
+  @Test
+  public void testFullOnTopNLongVirtualColumn()
```

Comment:
Thanks even i was thinking why the test case was modified. We could easily add a new one. 
🚀

Reply:
Yeah, it was an accident in the original version of the patch. I meant to add a new one.